app/common/azure_blob_storage_handler.py

```diff
@@ -0,0 +1,41 @@
+import os
+import uuid
+
+from azure.storage.blob import BlobServiceClient, ContentSettings
+from sentry_sdk import capture_exception
+
+
+def get_container_or_create_if_not_exist(name):
+    connection_string = os.environ.get("AZURE_STORAGE_CONNECTION_STRING")
+    blob_service_client = BlobServiceClient.from_connection_string(connection_string)
+    container = blob_service_client.get_container_client(name)
+    if container.exists:
```

Comment:
exists er en metode...
```suggestion
    if container.exists():
```